lib/ansible/modules/cloud/lxc/lxc_container.py

```diff
@@ -722,7 +723,7 @@ def _config(self):
 
         container_config_file = self.container.config_file_name
         with open(container_config_file, 'rb') as f:
-            container_config = f.readlines()
+            container_config = [to_text(line, errors='surrogate_or_strict') for line in f.readlines()]
```

Comment:
Note: It's probably faster to write this like this:
``` python
container_config = to_text(f.read(), errors='surrogate_or_strict').splitlines()
```

There's overhead from calling the text conversion functions so calling them only once should be quicker.

Reply:
Also, I'm not sure if it matters but you could use ```to_native()``` instead of ```to_text()``` if container_config should be byte strings on python2 and text strings on python3.  In a quick look at the following code I didn't see anything that would be a problem with text strings on python2 but I put that out there in case @cloudnull does see something.

Reply:
instead of using to_native, my first commit to just open the file with 'r' and write with 'w' might be cleaner. that way it will be bytes in py2 and text in py3. After reading the py3 docs assumed you want to always use to_text/to_bytes in those cases.